src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -54,16 +73,42 @@ export default {
 		};
 	},
 	async mounted() {
-		this.allFacets = await fetchLoanFacets(this.apollo);
+		this.allFacets = await fetchLoanFacets(this.apollo, true);
 		await this.fetchFilterData(this.flssLoanSearch);
-	},
-	computed: {
-		totalCount() {
-			return 0;
-		},
+		const { loans, totalCount } = await runLoansQuery(
+			this.apollo,
+			{ ...this.flssLoanSearch, ...this.loanSearchState },
+			FLSS_ORIGIN_CATEGORY
+		);
+		this.loans = loans;
+		this.totalCount = totalCount;
 	},
 	methods: {
-		updateQuickFilters() {
+		async updateQuickFilters(filter) {
+			if (filter.gender !== undefined) {
+				this.flssLoanSearch.gender = filter.gender;
+			} else if (filter.sortBy) {
+				this.flssLoanSearch.sortBy = filter.sortBy;
+			} else if (filter.country) {
+				this.flssLoanSearch.countryIsoCode = filter.country;
+			} else {
+				// We want to reset the flss paramaters for categories
+				delete this.flssLoanSearch.sectorId;
+				delete this.flssLoanSearch.tagId;
+				delete this.flssLoanSearch.activityId;
+				delete this.flssLoanSearch.themeId;
+				this.flssLoanSearch = {
+					...this.flssLoanSearch,
+					...filter
+				};
+			}
+			const { loans, totalCount } = await runLoansQuery(
```

Comment:
This `runLoansQuery` could be extracted into a method in this component, since this call and the results being stored, are the same in both cases in this file. It will help with maintenance, such as if the context changes.

Reply:
I'll approve this PR, but I recommend adding a follow up ticket to extract some identical code here.

Reply:
I actually think we will want to potentially use a different query depending on results from loan card work - but yes I will note this as a follow up for restructuring!